.htaccess

```diff
@@ -113,11 +113,9 @@ DirectoryIndex index.php index.html index.htm
   # If your site is running in a VirtualDocumentRoot at http://example.com/,
   # uncomment the following line:
   # RewriteBase /
-  RewriteBase /special-collections
```

Comment:
Do we need to keep this line?

Reply:
Yeah, we should probably remove that line. Thanks!